packages/expo-gl-cpp-legacy/cpp/UEXGL.h

```diff
@@ -37,47 +37,47 @@ typedef unsigned int UEXGLObjectId;
 // [JS thread] Create an EXGL context and return its id number. Saves the
 // JavaScript interface object (has a WebGLRenderingContext-style API) at
 // `global.__EXGLContexts[id]` in JavaScript.
-UEXGLContextId UEXGLContextCreate(JSGlobalContextRef jsCtx);
+UEXGLContextId UEXGLContextCreate__Legacy(JSGlobalContextRef jsCtx);
 
 #ifdef __cplusplus
 // [JS thread] Pass function to cpp that will run GL operations on GL thread
-void UEXGLContextSetFlushMethod(UEXGLContextId exglCtxId, std::function<void(void)> flushMethod);
+void UEXGLContextSetFlushMethod__Legacy(UEXGLContextId exglCtxId, std::function<void(void)> flushMethod);
 #endif
 
 #ifdef __APPLE__
 // Objective-C wrapper for UEXGLContextSetFlushMethod
 typedef void(^UEXGLFlushMethodBlock)(void);
```

Comment:
Only reason for most of those changes is that symbols from two libraries collided with each other, actual type does not exists as symbol inside binary

If not that `extern "C"` statement it might be used as part of the symbol name for `UEXGLContextSetFlushMethodObjC` but not as standalone symbol